dataframe-jdbc/src/main/kotlin/org/jetbrains/kotlinx/dataframe/io/db/DuckDb.kt

```diff
@@ -41,10 +41,7 @@ import org.duckdb.DuckDBResultSetMetaData
 import org.duckdb.JsonNode
 import org.jetbrains.kotlinx.dataframe.DataFrame
 import org.jetbrains.kotlinx.dataframe.io.DbConnectionConfig
```

Comment:
seems the refactoring broke a lot of KDoc references in this file. But we can also fix that once we redo DuckDb integration with new better types :D